ZeeHomeScreen/Classes/Components/Cell/CellViewController.swift

```diff
@@ -12,7 +12,7 @@ import ApplicasterSDK
     
     public var selectedModel: NSObject?
     
-    public var delegate: ComponentDelegate?
+    public weak var delegate: ComponentDelegate?
```

Comment:
this delegate had strong reference preventing memory of HomePluginZPZee5CellViewController being released

Reply:
was manifesting on scrolling HomeScreen Carousels horizontally.